plugins/polio/js/src/domains/Budget/BudgetProcess/validation.ts

```diff
@@ -1,37 +1,215 @@
+            .matches(
+                /^\d{4}-\d{2}-\d{2}$/,
+                formatMessage(MESSAGES.invalidDate),
+            ),
+        submitted_to_orpg_operations1_at_WFEDITABLE: string()
+            .nullable()
+            .optional()
+            .matches(
+                /^\d{4}-\d{2}-\d{2}$/,
+                formatMessage(MESSAGES.invalidDate),
+            ),
+        feedback_sent_to_rrt1_at_WFEDITABLE: string()
+            .nullable()
+            .optional()
+            .matches(
+                /^\d{4}-\d{2}-\d{2}$/,
+                formatMessage(MESSAGES.invalidDate),
+            ),
+        re_submitted_to_orpg_operations1_at_WFEDITABLE: string()
+            .nullable()
+            .optional()
+            .matches(
+                /^\d{4}-\d{2}-\d{2}$/,
+                formatMessage(MESSAGES.invalidDate),
+            ),
+        submitted_to_orpg_wider_at_WFEDITABLE: string()
+            .nullable()
+            .optional()
+            .matches(
+                /^\d{4}-\d{2}-\d{2}$/,
+                formatMessage(MESSAGES.invalidDate),
+            ),
+        submitted_to_orpg_operations2_at_WFEDITABLE: string()
+            .nullable()
+            .optional()
+            .matches(
+                /^\d{4}-\d{2}-\d{2}$/,
+                formatMessage(MESSAGES.invalidDate),
+            ),
+        feedback_sent_to_rrt2_at_WFEDITABLE: string()
+            .nullable()
+            .optional()
+            .matches(
+                /^\d{4}-\d{2}-\d{2}$/,
+                formatMessage(MESSAGES.invalidDate),
+            ),
+        re_submitted_to_orpg_operations2_at_WFEDITABLE: string()
+            .nullable()
+            .optional()
+            .matches(
+                /^\d{4}-\d{2}-\d{2}$/,
+                formatMessage(MESSAGES.invalidDate),
+            ),
+        submitted_for_approval_at_WFEDITABLE: string()
+            .nullable()
+            .optional()
+            .matches(
+                /^\d{4}-\d{2}-\d{2}$/,
+                formatMessage(MESSAGES.invalidDate),
+            ),
+        feedback_sent_to_orpg_operations_unicef_at_WFEDITABLE: string()
+            .nullable()
+            .optional()
+            .matches(
+                /^\d{4}-\d{2}-\d{2}$/,
+                formatMessage(MESSAGES.invalidDate),
+            ),
+        feedback_sent_to_orpg_operations_who_at_WFEDITABLE: string()
+            .nullable()
+            .optional()
+            .matches(
+                /^\d{4}-\d{2}-\d{2}$/,
+                formatMessage(MESSAGES.invalidDate),
+            ),
+        approved_by_who_at_WFEDITABLE: string()
+            .nullable()
+            .optional()
+            .matches(
+                /^\d{4}-\d{2}-\d{2}$/,
+                formatMessage(MESSAGES.invalidDate),
+            ),
+        approved_by_unicef_at_WFEDITABLE: string()
+            .nullable()
+            .optional()
+            .matches(
+                /^\d{4}-\d{2}-\d{2}$/,
```

Comment:
You should save this regex somewhere in a constant either in iaso or bluesquare-components